tests/49.t

```diff
@@ -0,0 +1,7 @@
+description: backspace
+keys: ab \b c \\n ## BACKSPACE ENTER
```

Comment:
You could incorporate this in `tests/03.t` instead of adding a new test:

```
keys: abb \177 \b c \\n ## BACKSPACE BACKSPACE ENTER
```

Don't forget to remove `tests/49.t` from `Makefile.am`.